Update Topics article (#3948)
GoogleChrome/developer.chrome.com: site/en/docs/privacy-sandbox/topics/index.md

## Test with `chrome://flags` or feature flags {: #feature-flags}

> There are two ways can try out the Topics API for a single user running Chrome 101 or above:

Reviewer: ```suggestion
There are two ways to try the Topics API as a single user, running Chrome 101 or above:
```

*  Enable `chrome://flags/#privacy-sandbox-ads-apis`
*  Run Chrome from the command line with the following flags: 

``` text
--enable-features=BrowsingTopics,PrivacySandboxAdsAPIsOverride,OverridePrivacySandboxSettingsLocalTesting
``` 

> The [Topics demo](#demo) shows how to use additional flags to adjust settings such as epoch 
> length. If you access the Topics API by running Chrome with command-line flags, don't 
> set `chrome://flags`, since these can override command-line settings. 

Reviewer: ```suggestion
The [Topics demo](#demo) shows how to use additional flags to adjust settings such as epoch 
length. If you access the Topics API by running Chrome with command-line flags, don't 
set `chrome://flags`, as these can override command-line settings. 
```
